### PR TITLE
reduce loss of precision from svec -> smat conversion

### PR DESCRIPTION
**Pull request by mlubin**
Thanks @blegat :)

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -335,7 +335,7 @@ OBJACOORD
 
 FCOORD
 1
-0 0 0 1 -0.9999999999999998
+0 0 0 1 -0.9999999999999999
```
Comment (blegat) [inline test/runtests.jl:338]: Why is it still not -1.0 ?

Comment (mlubin) [reply test/runtests.jl:338]: Dunno

Comment (mlubin) [reply test/runtests.jl:338]: @blegat, is it a precise statement that multiplication and division by sqrt(2) is an identity operation on ``Float64``s?

Comment (blegat) [reply test/runtests.jl:338]: No, you might loose precision. It only works for special cases like with 1.0.
For instance,
```julia
(pi * sqrt(2)) / sqrt(2) == pi # -> false
```
But here it seems you have -1.0 so I would have expected it to work

**Pull request merged**